app/views/layouts/_main_nav.html.erb

```diff
@@ -13,8 +13,8 @@
     visible based on the current state (see
     ApplicationHelper#display_navigation_link? for details).  %>
 <% if other_nav_links.any? %>
-  <nav class="mb-4" aria-labelledby="other-nav-heading">
-    <h2 id="other-nav-heading" class="crayons-subtitle-3 py-2 pl-3">
+  <nav class="mb-4 other-nav-heading">
```

Comment:
Hmm…removing the `aria-labelledby` does not strike me as something we can do.  I'm wondering if we want to pass a "prefix" in to this partial?

Reply:
I may disagree here because [nav are sections](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) so the `h2` is enough to me. I don't think `aria-labelledby` is that useful.

Reply:
@aitchiss could I get you to help us work through this?

Reply:
Thanks for tagging me in! I can confirm we do indeed want to keep the `aria-labelledby` to link the heading with the `nav`. On any given page we have multiple `navigation` landmarks; the impact of labelling each one is that instead of a screen reader user being presented with:

```
navigation
navigation
navigation
```

they will instead be presented with, e.g.:

```
main navigation
member menu navigation
other navigation
```

This allows screen reader user to go directly to the navigation element they want to interact with, without having to read all of the content in each one 🙂

Reply:
@aitchiss is it something I can apply every time I have 2 or 3 `<nav>` in the page to improve screen reader experience? For example, main navigation, secondary navigation, pagination.

Reply:
Yes definitely! As a general rule, it's best to start labelling `nav` elements as soon as you have more than one on a given page. It's also useful for `<aside>` elements if there's more than one on a page. It just helps screen reader users "scan" the page for which area they want to dip into 🙂 

When there's already a heading in the element, you can just link that in with `aria-labelledby`, but if not, `aria-label` also does the trick!

Reply:
@jeremyf I'm to pass some var in the render to make 2 different IDs.
@aitchiss thanks! TIL 🙏🏻